guide/mechanics/damageFormula.md

```diff
@@ -8,27 +8,35 @@ sidebar_label: Damage Formula
 
-**C:** Target Crit Multiplier For anything that cannot crit, this value is 1. Each weakpoint on an enemy has it's on crit multiplier. For most enemies, this value is 0.5. Some enemies such as scars, outlaw shotgunners, and Ash Titan's have higher or lower values.
+**TypeModifier:** This represents the damage bonus (or penalty) applied based on the damage type of your attack and the health type of the enemy. All attacks fall into one of 5 categories: Impact, Acid, Electric, Ice, or Fire. A table showing what the value of D should be is shown in the table below
 
-**D:** Critical Damage Modifiers.  If your item cannot crit, this value is 0. If your item can crit, this value is the total of all applicable crit damage modifiers. If you have 10% on your primary gun (gear icon) and 50% on your secondary gun (gear icon), these values are 0.1 and 0.5, respectively. However, some slots other than weapons can roll javelin-wide (javelin icon) critical inscriptions. These inscriptions should be included.
+**SumDebuffModifiers:** Sum of all debuffs applied to an enemy that cause them to take additional damage. Currently Acid, Interceptor's Target Beacon, and Colossus' Taunt. Beacon and Taunt do not stack with each other. Thus, the maximum value is 
 
-**E:** Damage type multiplier. This represents the damage bonus (or penalty) applied based on the damage type of your attack and the health type of the enemy. All attacks fall into one of 5 categories: Impact, Acid, Electric, Ice, or Fire. A table showing what the value of D should be is shown in the table below
+**ItemCritBonus:** For anything that cannot crit, this value is 1. For anything that can crit, this value depends on the damage source. Every weapon and ability that can crit has a crit multiplier specific to the item. Blastback is 2.4, Plasma Star is 1.5. a full list of datamined values can be found on [AnthemArchive](http:AnthemArchive.com) Please note that shielded enemies cannot be critically hit for damage purposes.
 
-**F:** Debuff multiplier. If the enemy is neither afflicted by Acid nor Target Beacon, this value is 1. This value increased by 0.25 if the enemy is afflicted by Acid, and increases by 0.33 if the enemy is marked with Target Beacon. If both are applied, this value is 1.58.
+**WeakPointMultiplier:** Target Crit Multiplier For anything that cannot crit, this value is 1. Each weakpoint on an enemy has it's on crit multiplier. For most enemies, this value is 0.5. Some enemies such as scars, outlaw shotgunners, and Ash Titan's have higher or lower values.
 
+**CritModifiers:**  If your item cannot crit, this value is 0. If your item can crit, this value is the total of all applicable crit damage modifiers. If you have 10% on your primary gun (gear icon) and 50% on your secondary gun (gear icon), these values are 0.1 and 0.5, respectively. However, some slots other than weapons can roll javelin-wide (javelin icon) critical inscriptions. These inscriptions should be included.
```

Comment:
Take out the first sentence and the start of the 2nd. Whether something can crit or not doesn't matter here. If it can't crit, the above `ItemCritBonus` which will be 0 and make this factor irrelevant. I would clarify the last part and just say "it is the sum of all gear and javelin wide modifiers that affect this item". Explaining how javelin vs gear works is out of scope here